README.md

```diff
@@ -139,7 +166,7 @@ export async function writeContent () {
 
```

Comment:
Same comment for the `options` object documentation above this - I can't put the comment directly on the line of code, but just prior to the `writeContent` example, there is a code block for the options. Can you put that in a block using backticks for consistency?